src/gc/gc.d

```diff
@@ -2910,6 +3117,8 @@ struct Pool
         auto nbits = cast(size_t)poolsize >> shiftBy;
 
         mark.alloc(nbits);
+        if(GC.config.precise)
+            is_pointer.alloc(cast(size_t)poolsize/(void*).sizeof);
```

Comment:
I just tried lazy allocation of Pool.is_pointer, but it makes most of the benchmarks run a little slower. I don't think there are many pools without pointers.